spec/mailers/notification_mailer_spec.rb

```diff
@@ -5,8 +5,8 @@
 RSpec.describe NotificationMailer do
   let(:receiver)       { Fabricate(:user, account_attributes: { username: 'alice' }) }
   let(:sender)         { Fabricate(:account, username: 'bob') }
-  let(:foreign_status) { Fabricate(:status, account: sender, text: 'The body of the foreign status') }
-  let(:own_status)     { Fabricate(:status, account: receiver.account, text: 'The body of the own status') }
+  let(:foreign_status) { Fabricate(:status, account: sender, text: 'The body of the foreign status', created_at: '2024-01-01 12:01Z') }
```

Comment:
Given the repeated use of same datetime values here ... could we pull them all out to private method at bottom of this spec? (and same in other spec?)

Reply:
I did this for the user_mailer_spec.rb.

In notification_mailer_spec.rb there are two different timestamps — one of them is referenced just once in the test body, and the other is referenced twice, so I don't think there is a lot duplication (compared to e.g. the name “bob” in the same file). Do you agree?. I have selected some new timestamps to make them more easily recognizable (2021-01-01 01:01 and 2022-02-02 02:02).